src/runtime/components/component.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <svg :view-box="viewBox" :class="icon.class">
```

Comment:
`:view-box="viewBox"` -> `:viewBox="viewBox"`

Reply:
The code is correct: https://v2.vuejs.org/v2/style-guide/?redirect=true#Prop-name-casing-strongly-recommended

Reply:
Does not work in Vue version 3 :(